<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formbarang</class>
 <widget class="QWidget" name="Formbarang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="id_barangLabel">
      <property name="text">
       <string>id_barang</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="id_barangLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaBarangLabel">
      <property name="text">
       <string>nama barang</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaBarangLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="spesipikasiLabel">
      <property name="text">
       <string>spesipikasi</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="spesipikasiLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="hargaSatuanLabel">
      <property name="text">
       <string>harga satuan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="hargaSatuanLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="simpan">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
